src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -1686,12 +1685,8 @@ export default class EthereumApi implements Api {
       this.#options
     );
 
-    if (isUnlockedAccount) {
-      const secretKey = wallet.unlockedAccounts.get(fromString);
-      return blockchain.queueTransaction(tx, secretKey);
-    } else {
-      return blockchain.queueTransaction(tx);
-    }
+    const secretKey = wallet.unlockedAccounts.get(fromString);
```

Comment:
Can we change the lines above: 
```
const isUnlockedAccount = wallet.unlockedAccounts.has(fromString);

if (!isUnlockedAccount) {
```
to 

```
const secretKey = wallet.unlockedAccounts.get(fromString);

if ( secretKey === undefined ) {
```

?

Reply:
🥲